WordPress/gutenberg
# [Mobile] Cover: Adds Clear media button in block settings

**Fixes** https://github.com/WordPress/gutenberg/issues/23789 (**parent** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2275)

## Related PRs:
* Gutenberg-Mobile https://github.com/wordpress-mobile/gutenberg-mobile/pull/2489
* Create cover block with only colour from a default palette https://github.com/WordPress/gutenberg/pull/23833
* Create cover block with custom colour https://github.com/WordPress/gutenberg/pull/23870
* Media Button when only colour is selected https://github.com/WordPress/gutenberg/pull/23878

## Description
Adds Clear media button in block settings

## How has this been tested?
* Add a cover block
* Press the *ADD IMAGE OR VIDEO* button and select a media
* Tap on the block settings
* Verify that there is a clear media button that removes the set image

## Screenshots

|With set media|Without set media|
|---|---|
|![Simulator Screen Shot - iPhone 11 - 2020-07-08 at 18 22 53](https://user-images.githubusercontent.com/304044/86938690-5ae5bb80-c149-11ea-93f9-7d1e93746824.png)|![Simulator Screen Shot - iPhone 11 - 2020-07-08 at 18 23 14](https://user-images.githubusercontent.com/304044/86938711-5faa6f80-c149-11ea-8d31-545d86804a50.png)|

## Types of changes
Enhancement

## Checklist:
